rpc-rs/src/chunkify.rs

```diff
@@ -140,24 +149,28 @@ impl ChunkEndpoint {
 
 pub(crate) fn chunkify_endpoint(
     domain: Option<String>,
+    nc: async_nats::Client,
     lattice: String,
 ) -> RpcResult<ChunkEndpoint> {
-    let js = connect_js(domain, &lattice)?;
+    let js = connect_js(domain, nc, &lattice)?;
     Ok(ChunkEndpoint::new(lattice, js))
 }
 
-pub(crate) fn connect_js(domain: Option<String>, lattice_prefix: &str) -> RpcResult<JetStream> {
+pub(crate) fn connect_js(
+    domain: Option<String>,
+    nc: async_nats::Client,
+    lattice_prefix: &str,
+) -> RpcResult<Context> {
```

Comment:
Can this ever fail now? If not, consider changing the return type to just `Context` (and then the return type of `chunkify_endpoint` could also be updated)